SQL.js

```diff
@@ -35,6 +35,20 @@ class SqlStatement {
     return new SqlStatement(result.strings, result.values)
   }
 
+  /**
+   * A function that accepts an array of objects and a mapper function
+   * It returns a clean SQL format using the object properties defined in the mapper function
+   */
+  map (array, mapFunc) {
+    if ((mapFunc instanceof Function) && array && array.length > 0) {
+      return this.glue(
+        array.map(mapFunc).map((item) => SQL`${item}`),
```

Comment:
as SQL should be a function, technically you can reduce it to:
```suggestion
        array.map(mapFunc).map(SQL),
```

Reply:
I tried it and the tests failed. I got undefined values instead of the actual IDs